setup.py

```diff
@@ -27,7 +27,7 @@
     description="Python 3.6+ toolbox for submitting jobs to Slurm",
     author="Facebook AI Research",
     python_requires=">=3.6",
-    long_description=Path("README.md").read_text(),
+    long_description=Path("README.md").read_text(encoding="utf-8"),
```

Comment:
@gwenzek that should be enough right? not too sure how to test

Reply:
yes this has been tested in #24